apps/vx-scan/frontend/src/components/calibrate_scanner_modal.tsx

```diff
@@ -23,12 +23,13 @@ export function CalibrateScannerModal({
   const calibrateMutation = calibrate.useMutation();
   const [calibrationState, setCalibrationState] =
     useState<CalibrationState>('ready');
-  const makeCancelable = useCancelablePromise();
 
-  async function onCalibrate() {
+  function onCalibrate() {
     setCalibrationState('calibrating');
-    const success = await makeCancelable(calibrateMutation.mutateAsync());
-    setCalibrationState(success ? 'calibrated' : 'failed');
+    calibrateMutation.mutate(undefined, {
```

Comment:
Nice - I like the recommendation of preferring `.mutate` over `.mutateAsync`

Out of scope for this PR of course but would be cool if we could write a lint rule to enforce the use of `.mutate` (or `.mutateAsync` wrapped in a try-catch)!

Reply:
Yes, a lint rule would be great! I'll save that for after getting feedback from the team on this approach